test/http2/test_http2_protocol.py

```diff
@@ -0,0 +1,216 @@
+
+import OpenSSL
+
+from netlib import http2
+from netlib import tcp
+from netlib import test
+from netlib.http2.frame import *
+from test import tutils
+
+
+class EchoHandler(tcp.BaseHandler):
```

Comment:
actually this one is broken.
No we can't reuse - because HTTP/2 does not work with `readline` because of missing line breaks.